pkg/cmd/marathon/marathon_ping.go

```diff
@@ -13,7 +15,19 @@ func newCmdMarathonPing(ctx api.Context) *cobra.Command {
 		Use:   "ping",
 		Short: "Ensure Marathon is up and responding.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return python.InvokePythonCLI(ctx)
+			client, err := marathon.NewClient(ctx)
+			if err != nil {
+				return err
+			}
+
+			_, err = client.API.Ping()
+			if err != nil {
+				return err
+			}
+
+			// If no err, a pong has been received.
```

Comment:
Why not use actual response body?

Reply:
The number appearing in `Marathon ping response[%dx]` is the number of times the ping requests was sent according to the Python logic.